index.html

```diff
@@ -3,11 +3,21 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>有限会社ヤタカ物産　建材各種販売</title>
     <link href="/css/reset.css"  rel="stylesheet" />
     <link href="/css/style.css"  rel="stylesheet" />
```

Comment:
Noto Sans JP を使えるように下記を追加ください🙏（説明できてなくてごめん）
Webフォントの使い方も、のちほど口頭でレクチャーします！
```suggestion
    <link href="/css/style.css"  rel="stylesheet" />
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@100..900&display=swap" rel="stylesheet">
```